source/common/grpc/grpc_web_filter.cc

```diff
@@ -130,5 +168,47 @@ Http::FilterTrailersStatus GrpcWebFilter::encodeTrailers(Http::HeaderMap& traile
   return Http::FilterTrailersStatus::Continue;
 }
 
+void GrpcWebFilter::setupStatTracking(const Http::HeaderMap& headers) {
```

Comment:
Ultimately I would like to just deprecate the http11 bridge filter once we get grpc-web fully rolled out. Not sure how long that will take. If you think it's worth it to do super class stuff that's fine with me.